src/Layouts/Partials/FillsBoxPanel.vala

```diff
@@ -58,7 +58,8 @@ public class Akira.Layouts.Partials.FillsBoxPanel : Gtk.Grid {
         add_btn.add (new Gtk.Image.from_icon_name ("list-add-symbolic",
                                                    Gtk.IconSize.SMALL_TOOLBAR));
         add_btn.clicked.connect (() => {
-            fills_list_model.add ();
+            //TODO: What means add another fillitem to an object: gradient???
```

Comment:
@Alecaddd ^^^

Reply:
@bilelmoussaoui Alex share some info about that. 

![fillings_cut](https://user-images.githubusercontent.com/220968/66750121-e61c7400-ee8b-11e9-9ff5-0e6f8fc38cd1.gif)

Implementation (and how it is useful) is clear to me with that